### PR TITLE
feat: change japanese language code from jp to ja

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA

https://disky.console.doodle.spaceone.dev/

### CODE REVIEW

File: apps/web/src/common/components/layouts/PdfDownloadOverlay/PdfDownloadOverlay.vue [hunk 1]
```diff
@@ -174,9 +174,9 @@ export default defineComponent<Props>({
             isMakingPdfStarted: false,
             progressRate: 0,
             font: computed<PdfFontFamily>(() => {
-                if (fontLanguages.en === props.fontLanguage) return pdfFontFamily.NotoSans;
+                if (fontLanguages.ja === props.fontLanguage) return pdfFontFamily.NotoSansJp;
                 if (fontLanguages.ko === props.fontLanguage) return pdfFontFamily.NotoSansKo;
-                return pdfFontFamily.NotoSansJp;
+                return pdfFontFamily.NotoSans;
```
Comment (WANZARGEN) [inline apps/web/src/common/components/layouts/PdfDownloadOverlay/PdfDownloadOverlay.vue:179]: description: Fixed a potential bug. English must be the fallback language.

File: apps/web/src/translations/vendors/dayjs.ts [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
+    ja: () => import('dayjs/locale/ja'),
 };
 
 const cachedDayjsLocales: Record<string, boolean> = {};
 
-export const dayjsLocaleMap = {
-    en: 'en',
-    ko: 'ko',
-    jp: 'ja',
-};
```
Comment (WANZARGEN) [inline apps/web/src/translations/vendors/dayjs.ts:13]: description: Removed unnecessary object. It was generated to reduce confusion which from different code names, jp and ja.

File: apps/web/src/store/modules/user/index.ts [hunk 3]
```diff
@@ -20,7 +22,7 @@ const state: UserState = {
     backend: storedUserState.backend,
     name: storedUserState.name,
     email: storedUserState.email,
-    language: storedUserState.language || 'en',
+    language: (storedUserState.language && languages[storedUserState.language]) ? storedUserState.language : 'en',
```
Comment (WANZARGEN) [inline apps/web/src/store/modules/user/index.ts:25]: description: This is the conversion code which makes work with both, 'ja' and 'jp'.

File: apps/web/src/store/modules/user/actions.ts [hunk 4]
```diff
@@ -17,7 +19,7 @@ const getDomainOwnerInfo = async (ownerId: string): Promise<Partial<UserState>>
         backend: 'LOCAL',
         name: response.name,
         email: response.email,
-        language: response.language,
+        language: languages[response.language] ? response.language : 'en',
```
Comment (WANZARGEN) [inline apps/web/src/store/modules/user/actions.ts:22]: description: This is the conversion code which makes work with both, 'ja' and 'jp'.

**Review by WANZARGEN — COMMENTED** [review]
I supplemented some descriptions.

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Pull request merged**